src/Microsoft.Health.Dicom.Functions/Indexing/Configuration/ReindexConfiguration.cs

```diff
@@ -39,5 +41,13 @@ public class ReindexConfiguration
         ///// </summary>
         //[Range(0d, 1d)]
         //public double MaxErrorPercentage { get; set; } = 1;
+
```

Comment:
These values should be in a different configuration section as they do not necessarily concern "reindexing"

Reply:
By creating a new section, you'll need to configure it like the existing `ReindexConfiguration` in *Startup.cs*